package.json

```diff
@@ -1,9 +1,6 @@
 {
   "name": "@clinic/flame",
   "version": "11.0.3",
-  "engines": {
-    "node": ">= 12.13.0"
-  },
```

Comment:
why remove it altogether rather than update it?

Reply:
Not really a strong reason. But, I tend to remove the engines field because it makes the backporting support difficult. We don't have LTS versions so it wouldn't be a problem. But, just for reference: https://github.com/fastify/fastify/issues/3163

Do have an objection to removing it?

Reply:
no